EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs

```diff
@@ -78,11 +81,67 @@ public static void ReportBug()
             Application.OpenURL("https://github.com/mobiledgex/edge-multiplay-unity-client/issues/new/choose");
         }
 
-        [MenuItem("EdgeMultiplay/Version 1.1", false, 80)]
+        [MenuItem("EdgeMultiplay/Server Stats", false, 80)]
+        public static async void ServerStats()
+        {
+            if (EditorUtility.DisplayDialog("EdgeMultiplay", "Where are you are running the server?", "Locally", "MobiledgeX"))
+            {
+                Application.OpenURL("http://localhost:7776");
+            }
+            else
+            {
+                string fqdn = await GetMobiledgeXAppFQDN();
+                if (fqdn == "")
+                {
+                    Debug.LogError("Make sure MobiledgeX Settings AppDefs and Region are correct, Check the console for more details.");
+                }
+                else
+                {
+                    Application.OpenURL("http://" + fqdn + ":7776");
+                }
+            }
+        }
+
+        [MenuItem("EdgeMultiplay/Version 1.2", false, 80)]
         public static void Version()
         {
             //placeholder for version number
         }
+
+#endregion
+
+#region  Helper functions
+
+        public static async Task<string> GetMobiledgeXAppFQDN()
+        {
+            MobiledgeXIntegration integration = new MobiledgeXIntegration();
+            try
+            {
+                await integration.RegisterAndFindCloudlet();
+                if (integration.FindCloudletReply.status == FindCloudletReply.FindStatus.FIND_FOUND)
+                {
+                    return integration.FindCloudletReply.fqdn;
+                }
+                else
+                {
+                    return "";

```

Comment:
hmm. If you return an empty string, the OpenURL will just get a 404 error right? Should probably log this at least

Reply:
In the code, I check and Log the error already

Reply:
Gotcha. I see that you tell them to check console and you have logs for RegisterClientException and FindCloudletException. Could you log the FindCloudletStatus here as well then?